src/deepsparse/tasks.py

```diff
@@ -222,14 +230,23 @@ def is_text_generation(cls, task: str) -> bool:
             for text_generation_task in cls.text_generation
         )
 
+    @classmethod
+    def is_chat(cls, task: str) -> bool:
+        """
+        :param task: the name of the task to check whether it is a chat task,
+        such as chatbot
+        :return: True if it is a chat task, False otherwise
+        """
+        return any(chat_task.matches(task) for chat_task in cls.chat)
+
     @classmethod
     def is_nlp(cls, task: str) -> bool:
         """
         :param task: the name of the task to check whether it is an nlp task
             such as question_answering
         :return: True if it is an nlp task, False otherwise
         """
-        return any([nlp_task.matches(task) for nlp_task in cls.nlp])
+        return any(nlp_task.matches(task) for nlp_task in cls.nlp)
```

Comment:
These nits should be a part of a separate PR ideally, but added them here as I was already touching this file